### DISCUSSION TITLE
Disparition mystérieuse

### DISCUSSION THREAD

**Discussion by MymiF** [marian-m12l/studio#479] [Q&A]
Tout d'abord merci à toute la communauté pour votre travail.
Je me permets de vous exposer mon problème  : 
J'ai une lunii dernière génération (V3). J'ai installé la version 0.5.1 de Studio. J'ai créé mon propre pack d'histoire en m'enregistrant. 
Ma difficulté c'est que les histoires que je mets sur la Lunii disparaissent !! La première fois au bout de quelques jours. Aujourd'hui au bout de quelques minutes d'écoute !!!!
Je ne trouve pas de réponse à cette difficulté et je n'ai pas accès au discord (pas d'invitation récente).
Je vous remercie pour l'aide que vous pourrez m'apporter.
Bonne journée!

**Comment by marian-m12l** [comment]
Bonjour @MymiF 
La dernière version sur ce dépôt est la 0.4.1, votre version provient certainement d'un fork de STUdio ?
Difficile de vous aider dans ces conditions, mais si je comprends bien le pack d'histoire est bien reconnu et la lecture fonctionne pendant un certain temps ? C'est le cas pour différents packs ?

**Reply by MymiF** [reply]
Merci pour votre réponse.
Je croyais bêtement que la version 0.5 était un upgrade de la version 0.4 !

J'ai intégré, vendredi, mon pack d'histoires à l'aide de la version 0.4.1 de STUDIO. Ma fille en a écouté pendant le week-end mais ce matin (lundi), à l'allumage, je m'aperçois que le pack n'est plus dans la lunii :-(
La Lunii n'est pas sensée se connecter au wifi sans manipulation de notre part mais est-ce vraiment le cas?

**Reply by marian-m12l** [reply]
La lecture fonctionne sans erreur jusqu'au bout ? J'imagine que la lunii peut désactiver un pack d'histoires en cas d'erreur.
Quoi qu'il en soit, l'appareil est tout à fait en mesure de détecter un pack non-officiel, mais je ne sais pas si une telle restriction existe.

**Reply by MymiF** [reply]
Je réponds un peu tard mais je confirme que oui, tout se passe très bien et d'un coup la lunii supprime mes histoires.
... mais ça c'était avant! ....
J'ai supprimé l'appli de mon téléphone et ce problème n'apparaît plus. Donc je pense qu'effectivement la lunii se connecte à l'appli et vérifie le contenu.

Par contre j'ai maintenant un nouveau message d'erreur sur STUDIO : "Echec de récupération des packs d'histoires de l'appareil". Empêchant, par la même, de déposer nos histoires.

**Comment by MymiF** [comment]
J'ai l'impression que Lunii ajoute un applicatif au moment d'une nouvelle synchronisation qui fait planter la Lunii dès qu'elle se connecte à STUDIO. Après il ne nous reste plus qu'à la réinitialiser !!!! :-(((((((

**Comment by marian-m12l** [comment]
@MymiF Est-ce qu'il y a une erreur dans les logs ? Il est possible que Lunii ait déployé une évolution mal prise en charge par STUdio. L'appareil ne fonctionne plus après avoir transféré un pack d'histoires depuis STUdio ?

**Comment by Mimouss56** [comment]
Je me permets de compléter les informations que j'ai également de mon côté. Je viens d'avoir exactement le même soucis
Chargement de 15 pack d'histoire sans soucis, App de Lunii désinstaller aussitôt que celle ci fut opérationnelle.
Ce matin les histoires étaient encore, présente mais ce soir plus que les histoires achetées via le site. J'ai essayé de trouvé les log dans la Lunii mais rien.
Je viens de renommer le fichiers wifi.prefs en bak.wifi.prefs.bak pour voir si ça saute encore

**Reply by Mimouss56** [reply]
Depuis la suppression de ce fichier les histoires restent en place
